doc/nrf/ug_thingy53.rst

```diff
@@ -0,0 +1,258 @@
+.. _ug_thingy53:
+
+Working with Thingy:53
+######################
+
+.. contents::
+   :local:
+   :depth: 2
+
+Nordic Thingy:53 is a battery-operated prototyping platform for IoT Systems.
+
+Thingy:53 integrates the following components:
+
+* nRF5340 SoC - Supporting Bluetooth® Low Energy, 802.14.5 based protocols and Near Field Communication (NFC).
+* nRF21540 RF FEM - Range extender with integrated power amplifier (PA)/low-noise amplifier (LNA).
+* nPM1100 PMIC - Power Management IC with integrated dual-mode buck regulator and battery charger.
+
+For more information on the product, see the `Thingy:53 product page`_ and the `Nordic Thingy:53 User Guide`_.
+The |NCS| provides support for developing applications on the Thingy:53.
+
+.. _thingy53_serialports:
+
+Connecting to Thingy:53 serial ports
+************************************
+
+For connecting to Thingy:53, use a serial terminal.
+
+Thingy:53 uses the following UART baud rate configuration:
+
+.. list-table::
+   :header-rows: 1
+   :align: center
+
+   * - UART Interface
+     - Baud Rate
+   * - UART
+     - XXXXXXX
+
+
+Firmware
+********
+
+Nordic Thingy:53 comes preprogrammed with the Edge Impulse firmware, which allows the Thingy:53 to connect to Edge Impulse Studio over Bluetooth LE using the nRF Edge Impulse mobile application or through USB on your PC.
+The firmware allows you to capture sensor data, which in turn you can be use to train and test machine learning models in Edge Impulse Studio.
+You can deploy trained machine learning models to the Nordic Thingy:53 through an over-the-air (OTA) update using Bluetooth LE or update over USB using nRF Connect Programmer.
+
+See the Nordic Semi Thingy:53 page in the Edge Impulse documentation on how to use the Nordic Thingy:53 with the nRF Edge Impulse mobile application and Edge Impulse Studio.
+
+There are also multiple samples in the |NCS| that you can run on the Nordic Thingy:53.
+
+.. _thingy53_precompiled_fw:
+
+Programming precompiled firmware images
+***************************************
+
+Precompiled firmware image files are useful in the following scenarios:
+
+* Restoring the firmware to its initial image.
+* Updating the application firmware to a newly released version.
+* Testing different features of the Nordic Thingy:53.
+
+Downloading precompiled firmware images
+=======================================
+
+To obtain precompiled images for updating the firmware, perform the following steps:
+
+1. Go to the `Thingy:53 product page`_ and under the :guilabel:`Downloads` tab, navigate to :guilabel:`Precompiled application and modem firmware`.
```

Comment:
```
and modem firmware
```
?

Reply:
Should be "Precompiled applications".

Reply:
So there is no "and modem firmware" part?

Reply:
Do you mean `network core firmware`?

Reply:
I mean, is the section title "Precompiled applicatons and modem firmware", or just "Precompiled applications".

Reply:
> So there is no "and modem firmware" part?

Thingy53 has no modem so there cannot be a modem FW. There is a network core but this work differently. For starters each app can have different version of FW for net core.